### ISSUE TITLE
Add recognition to sample cards

### ISSUE THREAD

**Issue by pkbullock** [Adam-it/pnp-powershell-extension#5]
Hi @Adam-it 

This is awesome work on the extensions. If I may offer a suggestion, can you add some mention in the samples or the preview who the author was? Its quite important that we recognize those that have spent time creating these samples. 

![image](https://user-images.githubusercontent.com/8781041/189474655-2b5c8c10-57b7-4bb4-89db-ed3e415dfec5.png)

**Comment by Adam-it** [comment]
Awesome idea and TBH I already had that in mind when I created the CLI VS Code extension. 
The problem is that from the sample.json it is not clear who added the sample for PnP PowerShell and who added the CLI for M365 version. The extension targets only single tech. Either PnP PS or CLI and it shows only samples which have examples in that tech. So it's currently impossible to get the information in easy way who is the author of that sample in the specified tech 🤔.
What we could do is always show all authors, does not matter if one of them actually did the PnP PS version and not the CLI one. Do you think this would be fine? 
If not then maybe we could add this info somehow to the sample.json 🤔?

**Comment by Adam-it** [comment]
@pkbullock any comments on this topic ? 🤔

**Comment by pkbullock** [comment]
Best approach: Just all them all, if you are reading from sample.json files from PnP Scripts, then add last to first from the authors array.


Bad approach:
There is possibly a way (horrible and scary), each time a sample is added, under the metadata, it lists the tool e.g CLI, when a script on the same sample is added, I add the related tool at the end of the metadata array. When this is a new author, I PREPEND (due to a bug in the card display, you need to add author at the beginning of the authors array) the author for the sample. Its not clean, but you can calculate what they submitted.

**Comment by pkbullock** [comment]
e.g. 
![image](https://user-images.githubusercontent.com/8781041/192386697-e8e63da9-8c5a-4c10-9659-567d86505a3b.png)
![image](https://user-images.githubusercontent.com/8781041/192386754-e06d6271-85ef-4a62-a90a-1ea1bfa0e1e3.png)

**Comment by pkbullock** [comment]
One more thing, the "POWERSHELL" tag - do you need it? If not id save the real estate on the cards. 
The UI is nice and clean presentation btw - really like that :-)

**Comment by Adam-it** [comment]
ok I will list all authors 👍. I guess the more people we mention the better 🙂. 
totally not need the powershell tag 💪. the same 'card' react component is actually used in the CLI for M365 extension and there it may be either `posh` or `bash` so the tags have some value there 🙂. But since I already extracted this to a separate component, I will remove the tag in the PowerShell extension. Thanks for this additional point 👍.

To sum up:
- [x] list all authors
- [ ] remove file language tag

Adding this issue to the project plan to be done next time 🤩
